contents/docs/user-guides/experimentation.mdx

```diff
@@ -0,0 +1,198 @@
+
+While peeking at the results in itself is not a problem, making quick decisions based on preliminary results is problematic. Thus, we tell you when it's safe to end an experiment. When it's not safe, there's a red banner on top telling you it's not safe to end this experiment.
+
+Note that this recommendation doesn't cover all edge cases. It is still possible, when new data comes in, that results flip completely. This is true for any experiments and any significance test in existence today. But, the probability of this happening is very low, which is why most people are comfortable taking this risk. For a 0 risk experiment, you'd have to run it for an infinite amount of time: every new person is new data to be accomodated.
+
+### Terminating an experiment
+
+You decide when to terminate an experiment. A banner in your experiment results page will let you know if it's safe to terminate an experiment or not (regardless of the original estimated running time). If you terminate the experiment before you see this banner, results will not be significant (see Advanced for more details on this) and any conclusions could be completely wrong.
+
+Once you have decided to terminate an experiment there are a few things to do:
+
+-   [ ] Click on "Terminate the experiment" on the experiment page. This will ensure final results are kept.
+-   [ ] We recommend you roll-out the winning variant to all your users. TODO: Explain here how to do this with PostHog.
+-   [ ] Share results with relevant members of your team.
+-   [ ] After the winning variant is rolled-out, it's good practice to remove the other variants from your code, and make the winning variant part of your core code (i.e. stop checking for the respective Feature Flag).
+-   [ ] Document conclusions and findings in your PostHog experiment. This will help preserve context for team members who in the future need to understand what happened.
+-   [ ] Archive the experiment. This will terminate the Feature Flag and ensure your code no longer depends on the flag.
+
+## Advanced: What's under the hood?
+
+Below are all formulas and calculations we go through when recommending sample sizes and determining significance.
+
+### How we ensure distribution of people
+
+For every experiment, we leverage PostHog's multivariate feature flags. There's one control group, and up to 3 test groups. Based on their distinctID, each user is randomly distributed into one of these groups. This is stable, so the same users, even when they revisit your page, stay in the same group.
```

Comment:
How would this work with groups/orgs?

Reply:
good question, it doesn't yet (but it ought to)